TestON/drivers/common/cli/onosclidriver.py

```diff
@@ -1758,11 +1758,19 @@ def checkIntentState( self, intentsId, expectedState = 'INSTALLED' ):
             returnValue = main.TRUE
             for intents in intentsDict:
                 if intents.get( 'state' ) != expectedState:
-                    main.log.info( self.name + " : " + intents.get( 'id' ) +
-                                   " actual state = " + intents.get( 'state' )
-                                   + " does not equal expected state = "
-                                   + expectedState )
-                    returnValue = main.FALSE
+                    if intents.get( 'state' ) == "INSTALLING":
```

Comment:
I'm not sure this is what we actually want this function to do. 

As it is, we give it the expected state, and return true if the intent is in the expected state or INSTALLING state. This means if we expect the state to be WITHDRAWN the function returns true if the state is WITHDRAWN or INSTALLING. These two states are on opposite ends of the state machine.

What we may want to do is pass in a list of states to expect.

Also, in your test cases, you should be checking the the intent does eventually get into the INSTALLED state and not just the *ING states.